src/extrainterpreters/piped_interpreter.py

```diff
@@ -41,8 +41,8 @@ class FuncData(StructBase):
 def _dispatcher(pipe, buffer):
```

Comment:
this is in a very early stage - but the concept looks fine: a looping listener function in each interpreter tied to a queue which will send different tasks - which could be run a function, spin a thread, and so on.

I am thinking of doing away with the "PipedInterpreter" altogether, and have an optional post-start call which would start such a listenner function. The InterpreterPoolExecutor would do it automatically. 

Just let me know if you have a different  idea to approach this.